spec/turn_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/card'
+require './lib/turn'
+
+
+RSpec.describe Turn do
+  card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+  turn = Turn.new("Juneau", card)
+
+  it 'exists' do
+    expect(card).to be_a(Card)
```

Comment:
We already made this check from inside the `card_spec.rb` file so doing it again here is just redundant.